### DISCUSSION TITLE
Compile Textadept As An APK

### DISCUSSION THREAD

**Discussion by abakasam** [orbitalquark/textadept#74] [Ideas]
May you compile the next release as an APK?

Sam

Current app I am using.
![Screenshot 2021-03-13 5 25 08 PM](https://user-images.githubusercontent.com/58202540/111052529-1a99b400-8421-11eb-9142-749709cf4fbc.png)

**Comment by orbitalquark** [comment]
Is that for ARM? If so, I cannot because I don't have an ARM cross-compiling toolchain set up. Sorry :(

**Comment by abakasam** [comment]
What I saw might be possible, not sure.

Compiling Textadept with Docker
https://reposhub.com/cpp/miscellaneous/orbitalquark-textadept.html

Using Docker to cross compile
https://blog.kitware.com/cross-compiling-for-android-with-docker/
https://laptrinhx.com/compiling-android-source-code-in-docker-environment-compiling-runnable-xposed-1644039589/
http://blog.thenextgenlearn.com/2021/03/09/simple-step-to-build-an-android-app-with-docker-and-use-it-in-continuous-integration/#gs.w3mw7v
https://developer.toradex.com/knowledge-base/how-to-setup-android-build-environment-using-docker

Sam

**Comment by orbitalquark** [comment]
Thanks for the info. When I have some time, I may investigate this, but I have no way to test if it ends up working.
